src/coreclr/inc/corinfo.h

```diff
@@ -874,24 +874,15 @@ enum CorInfoException
     CORINFO_Exception_Count,
 };
 
-
-// This enumeration is returned by getIntrinsicID. Methods corresponding to
-// these values will have "well-known" specified behavior. Calls to these
-// methods could be replaced with inlined code corresponding to the
-// specified behavior (without having to examine the IL beforehand).
-
-enum CorInfoIntrinsics
+// These are used to detect array methods as NamedIntrinsic in JIT importer,
+// which otherwise don't have a name.
+enum class CorInfoArrayIntrinsic : UINT8
```

Comment:
```suggestion
enum class CorInfoArrayIntrinsic
```
Nit: The default backing is just fine for enums on the JIT/EE interface.